app/mailers/signup/abo_magazin_mailer.rb

```diff
@@ -24,11 +24,8 @@ def confirmation(person, group, newsletter_subscribed)
   private
 
   def placeholder_costs
-    cost = if @person.country == "CH"
-      SacCas::ABO_COSTS[:magazin].find { |cost| cost.country == :switzerland }
-    else
-      SacCas::ABO_COSTS[:magazin].find { |cost| cost.country == :international }
-    end
+    cost = Group.root.abo_alpen_fee
+    cost += Group.root.abo_alpen_postage_abroad if @person.living_abroad?
```

Comment:
In any other place where abroad fees are applied, we use living_abroad? to to not apply costs to Liechtenstein, this should probably be the same here :smile: